e2e/testIDs.js

```diff
@@ -3,11 +3,17 @@ const testIDs = {
 		tacView: 'tac_view',
 		agreeTacButton: 'tac_agree',
 		agreePrivacyButton: 'tac_privacy',
+		nativeModuleTestButton: 'tac_native_test',
 		nextButton: 'tac_next'
 	},
 	AccountListScreen: {
-		accountList: 'accountList',
-	}
+		accountList: 'accountList'
+	},
+	NativeTestScreen: {
```

Comment:
Is this just a dummy-UI for testing non-UI features?

Reply:
yeah, it is a dummy UI only display on the development environment. 

I have not an expert to do the testing on Native modules. I thought about how to test about them, the reason I choose to use E2E for testing it is:
1. a central place for testing all the native modules together.
2. one command could run the test without mouse interactions.

If there is a native way to fulfill these goals, then I am happy change to that :)